.env.sample

```diff
@@ -11,3 +11,8 @@ BPP_URI=
 
 # Provider UI
 PROVIDER_UBA_UI_URL=
+
+
+BENEFITS_BATCH_NUM=
```

Comment:
rename and use as

BENEFIT_CALCULATIONS_BATCH_SIZE
BENEFIT_CALCULATIONS_LAST_PROCESS_HOURS
BENEFIT_CALCULATIONS_CRON_TIME